exchange/cache.go

```diff
@@ -80,6 +82,26 @@ func GetServicePolicyFromCache(svcOrg string, svcId string, svcArch string, svcV
 	return nil
 }
 
+// GetServicePolicyFromCache returns the service policy from the exchange cache if it is present, or nil if it is not
```

Comment:
copy/paste problem in the comments, referencing service policy.

Reply:
Fixed